cmd/infracost/testdata/breakdown_invalid_apikey/breakdown_invalid_apikey.golden

```diff
@@ -0,0 +1,5 @@
+
+Err:
+Error: Invalid API Key, please check your INFRACOST_API_KEY environment variable or /Users/hugorut/.config/infracost/credentials.yml credentials file.
```

Comment:
Is this going to fail for people who aren't called hugorut :trollface:?

Reply:
ah yes 🙃